python_modules/dagster/dagster/components/lib/executable_component/component.py

```diff
@@ -88,6 +90,7 @@ class ExecutableComponent(Component, Resolvable, Model):
 
     # inferred from the function name if not provided
     name: Optional[str] = None
+    tags: Optional[dict[str, Any]] = None
```

Comment:
should these be distinguished as `op_tags` since the specs can have `tags` as well?

Reply:
i guess to run through it
* `tags` - potentially confusing given `tags` on specs
* `op_tags` - aligns with decorator args, distinguishes from `tags` set directly on specs. partially surfaces notion of op
* `op: OpSpec` or similar - embrace op fully, structured so other `op_` prefix doesn't need to be repeated

Reply:
ooo `execution` 👍🏻

Reply:
mind if i do this at the tip of the stack?

Reply:
https://app.graphite.dev/github/pr/dagster-io/dagster/30434/RFC-ExecutionModel